docs/docs/end-users/VSCode.md

```diff
@@ -4,8 +4,26 @@ categoryindex: 1
 index: 8
 ---
 # Visual Studio Code
-TODO: Explain version of fantomas used in VSCode.</br>
-The recommended way to use Fantomas is by using the [Ionide plugin](http://ionide.io/). Fantomas is integrated in [FSAutoComplete](https://github.com/fsharp/FsAutoComplete/) which is the language server used by Ionide.
+The recommended way to use Fantomas in Visual Studio Code is by using the [Ionide plugin](http://ionide.io/). Fantomas is integrated in [FSAutoComplete](https://github.com/fsharp/FsAutoComplete/) which is the language server used by Ionide.  
+Make sure Ionide is set to the default formatter inside `settings.json` :
+
+```
+  "[fsharp]": {
+    "editor.formatOnSave": true,
+    "editor.defaultFormatter": "Ionide.Ionide-fsharp"
+  }
+```
+
+### Fantomas version detection used by FSAutoComplete
```

Comment:
I know I asked you to write this here, but this also applies to Visual Studio.
In short, change in the title `by FSAutoComplete` to [by Fantomas.Client](https://www.nuget.org/packages/Fantomas.Client). (No need to add the link, I'm just adding it here in the review comment)
The whole detecting fantomas version happens in there and is not tied to FsAutocomplete or the Visual Studio extensions.
Both FsAutocomplete and VS Extensions do use `Fantomas.Client`.